schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,264 @@
+
+| Key | Value | Annotator |
+:--|:--|:--
+| assay | `str` or categorical with `str` categories. This MUST be appended with any *clarifying text* set in matching identifier. | Data Portal |
+| cell_type | `str` or categorical with `str` categories. | Data Portal |
+| development_stage | `str` or categorical with `str` categories. This MUST be "unknown" if set in the matching identifier | Data Portal |
+| disease | `str` or categorical with `str` categories | Data Portal |
+| ethnicity | `str` or categorical with `str` categories. This MUST be "na" or "unknown" if set in the matching identifier. | Data Portal |
+| organism | `str` or categorical with `str` categories | Data Portal |
+| sex | `str` or categorical with `str` categories | Data Portal |
+| tissue | `str` or categorical with `str` categories. This MUST be appended with " (cell culture)" or " (organoid)" if set in the matching identifier. | Data Portal |
+| | | |
+
+## `var` (Gene Metadata)
+
+`var` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+`var.index` MUST contain unique ENSEMBL gene identifiers for features. 
+
+Curators MUST annotate the following columns in the `var` dataframe:
+
+| Key | Value | Annotator |
+:--|:--|:--
+| feature_biotype | This MUST be "gene". | Data Portal |
+| feature_id (`var.index`) | `str`. This MUST be an ENSEMBL term from the gene annotation corresponding to the organism for the gene.  | Curator |
+||||
+
+Curators SHOULD NOT annotate the following column in the `var` dataframe which is the human-readable name that MUST match the corresponding gene identifier. This column MUST be automatically annotated by the cellxgene Data Portal when a dataset is uploaded.
+
+| Key | Value | Annotator |
+:--|:--|:--
+| feature_name | `str`. This MUST be the ENSEMBL gene name corresponding to the `feature_id`. | Data Portal |
+||||
+
+## `obsm` (Embeddings)
+
+For each `str` key, `obsm` stores a `numpy.ndarray` of shape `(n_obs, m)`, where `n_obs` is the number of rows in `X` and `m >= 1`.
+
+To display a dataset in cellxgene Explorer, Curators MUST annotate **one or more** two-dimensional (`m >= 2`) embeddings (tSNE, UMAP, PCA, spatial coordinates) in `obsm`. The key for the embedding MUST be prefixed with `X_`. The text that follows this prefix is presented to users in the cellxgene Explorer *Embedding Choice* selector.
+
+To illustrate, the [Krasnow Lab Human Lung Cell Atlas, 10X dataset](https://cellxgene.cziscience.com/e/krasnow_lab_human_lung_cell_atlas_10x-1-remixed.cxg/) in the [A molecular cell atlas of the human lung from single cell RNA sequencing collection](https://cellxgene.cziscience.com/collections/5d445965-6f1a-4b68-ba3a-b8f765155d3a) defines two embeddings in `obsm`:
+
+* 'X_Compartment_tSNE'
+* 'X_tSNE'
```

Comment:
```suggestion
* `'X_Compartment_tSNE'`
* `'X_tSNE'`
```
I prefer references to code to use code formatting.

Reply:
Rewrote a bit. Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/a0909191d6f636f2c857ece47eae5537d1ef9223